### PR TITLE
move otel to a different profile

### PR DESCRIPTION
**Pull request by yagrxu**
Dears,

1. add a new profile `otel-manual` which contains only otel-manual required dependencies.
2. changed otel files to *.opt file
3. add script to rename files back to *.java

test result:
1. compile ok
2. file size from 67 -> 62 MB with command `ll -a`

_-rw-r--r--  1 yagrxu  staff    67M Jan 20 23:25 store-spring-1.0.0.jar
Vs
-rw-r--r--  1 yagrxu  staff    62M Jan 20 23:27 store-spring-1.0.0.jar_

**The change introduces incompatibility in workshop section /observability/others/ADOT Manual Instrumentation**

Best Regards
Yagr

### CODE REVIEW

**Comment by yagrxu** [comment]
@deki @ybezsonov @maschnetwork   Please review the change

File: labs/unicorn-store/software/unicorn-store-spring/pom.xml [hunk 1]
```diff
@@ -111,48 +111,103 @@
...
-            <groupId>io.opentelemetry.contrib</groupId>
-            <artifactId>opentelemetry-aws-xray-propagator</artifactId>
-            <version>1.32.0-alpha</version>
-        </dependency>
     </dependencies>
     <profiles>
+        <profile>
+            <id>otel-manual</id>
+            <build>
+                <plugins>
```
Comment (deki) [inline labs/unicorn-store/software/unicorn-store-spring/pom.xml:119]: Plugin definitions from the jvm profile shouldn't be duplicated here. Maven allows multiple profiles to be activated in parallel https://maven.apache.org/guides/introduction/introduction-to-profiles.html#Details_on_profile_activation

File: labs/unicorn-store/software/unicorn-store-spring/pom.xml [hunk 2]
```diff
@@ -111,48 +111,103 @@
...
+                                <image>public.ecr.aws/docker/library/amazoncorretto:17-alpine3.18</image>
+                            </from>
+                            <container>
+                                <user>1000</user>
+                            </container>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+            <dependencies>
```
Comment (deki) [inline labs/unicorn-store/software/unicorn-store-spring/pom.xml:165]: Can you integrate the change I proposed in #130 and use `opentelemetry-spring-boot-starter` to simplify dependency management?

**Review by deki — CHANGES_REQUESTED** [review]
Next time please separate whitespace/ formatting changes (makes it hard to see the real difference)

**Comment by yagrxu** [comment]
@deki  I move the dependency to opentelemetry-spring-boot-starter according to #130 and also combined the profile with jvm to build.

**Comment by yagrxu** [comment]
I will update the workshop with new content in manual ADOT section - it is already prepared and will be merged
 from feature branch to mainline

**Pull request merged**